qemu/tests/cfg/cdrom_block_size_check.cfg

```diff
@@ -20,3 +15,13 @@
         #umount_cdrom_cmd = "mountvol %s /d"
         #show_mount_cmd = "wmic volume list brief"
         check_cdrom_size_cmd = wmic LogicalDisk where drivetype=5 get size
+    variants:
+        - unlimited_iothread:
+            required_qemu = [7.0.0-4,)
+        - no_iothread:
+            required_qemu = [,7.0.0-3)
```

Comment:
should be `required_qemu = (,7.0.0-3]`

The current configuration will miss version 7.0.0-3
With 7.0.0-3:
```
 (1/2) Host_RHEL.m9.u1.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.1.0.aarch64.io-github-autotest-qemu.change_media.unlimited_iothread.arm64-pci: STARTED
 (1/2) Host_RHEL.m9.u1.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.1.0.aarch64.io-github-autotest-qemu.change_media.unlimited_iothread.arm64-pci: CANCEL: Got host qemu version:7.0.0-3, which is not in [7.0.0-4,) (2.46 s)
 (2/2) Host_RHEL.m9.u1.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.1.0.aarch64.io-github-autotest-qemu.change_media.no_iothread.arm64-pci: STARTED
 (2/2) Host_RHEL.m9.u1.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.1.0.aarch64.io-github-autotest-qemu.change_media.no_iothread.arm64-pci: CANCEL: Got host qemu version:7.0.0-3, which is not in [,7.0.0-3) (2.44 s)
```

Reply:
> should be `required_qemu = (,7.0.0-3]`
> 
> The current configuration will miss version 7.0.0-3 With 7.0.0-3:
> 
> ```
>  (1/2) Host_RHEL.m9.u1.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.1.0.aarch64.io-github-autotest-qemu.change_media.unlimited_iothread.arm64-pci: STARTED
>  (1/2) Host_RHEL.m9.u1.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.1.0.aarch64.io-github-autotest-qemu.change_media.unlimited_iothread.arm64-pci: CANCEL: Got host qemu version:7.0.0-3, which is not in [7.0.0-4,) (2.46 s)
>  (2/2) Host_RHEL.m9.u1.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.1.0.aarch64.io-github-autotest-qemu.change_media.no_iothread.arm64-pci: STARTED
>  (2/2) Host_RHEL.m9.u1.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.1.0.aarch64.io-github-autotest-qemu.change_media.no_iothread.arm64-pci: CANCEL: Got host qemu version:7.0.0-3, which is not in [,7.0.0-3) (2.44 s)
> ```

updated